CHANGELOG.md

```diff
@@ -54,10 +54,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - `@cumulus/message/workflows/getMessageWorkflowStartTime`
     - `@cumulus/message/workflows/getMessageWorkflowStopTime`
     - `@cumulus/message/workflows/getMessageWorkflowName`
+  - **CUMULUS-2199**
+    - Added `translateApiRuleToPostgresRule` to `@cumulus/db` to translate API Rule to conform to Postgres Rule definition.
 
 ### Changed
 
 - **CUMULUS-2185** - RDS Migration Epic
+  - **CUMULUS-2199**
+    - Updated Rules endpoint to write to and delete from RDS if a write to and delete from DynamoDB is successful.
```

Comment:
The logic is actually the opposite: the Dynamo write/delete is only attempted if the RDS write/delete is sucessful.
However I don't think the CHANGELOG needs to spell out the exact implementation logic, so I think we can simplify to

```suggestion
    - Updated Rules endpoint to write rules to core PostgreSQL database in addition to DynamoDB and to delete rules from the PostgreSQL database in addition to DynamoDB
```